upsertGitHubTag-f79c4f36-3e61-43f4-8f6c-0b2e0e9774d7/deployment/index.js

```diff
@@ -46,8 +46,8 @@ function callEndpoint(path, postBody, callback) {
 
        res.on('end', function() {
            if (callback) {
+               // If content-type is text/plain, the body contains the message, else the message is found in the JSON object
```

Comment:
👍

Reply:
@agduncan94 Ummm, I don't think that was my comment, I found it when I got there